### PR TITLE
NAV-40 Improve configs.

### PR DESCRIPTION
**Pull request by jonathansberry**
Just some improvements to the config structure.

### CODE REVIEW

File: navigator_engine/model.py [hunk 1]
```diff
@@ -20,7 +20,6 @@ def to_networkx(self):
 
 class Conditional(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    ref = db.Column(db.String)
```
Comment (jonathansberry) [inline navigator_engine/model.py:23]: @jsoppela Please confirm that this column is not actually needed in the model?

File: navigator_engine/app.py [hunk 2]
```diff
@@ -14,10 +14,12 @@ def create_app(config_object=None):
     app = Flask(__name__)
 
     if not config_object:
-        config_object = os.getenv('CONFIG_OBJECT', 'navigator_engine.config.Config')
+        config_object = os.environ.get(
+            'NAVIGATOR_ENGINE_SETTINGS',
```
Comment (tomeksabala) [inline navigator_engine/app.py:18]: I suggest keeping more standard env var name for the flask config object `CONFIG_OBJECT`

Comment (jonathansberry) [reply navigator_engine/app.py:18]: I have... https://flask.palletsprojects.com/en/2.0.x/config/#configuring-from-python-files

File: navigator_engine/model.py (hunk 1 above)
Comment (jsoppela) [reply navigator_engine/model.py:23]: We are not using the ref column in the database at the moment but the data frame in graph loader uses it as index column. I think storing it is still a good idea.

**Pull request merged**